coffee_core/src/coffee.rs

```diff
@@ -496,6 +499,16 @@ impl PluginManager for CoffeeManager {
                     let mut actions = self.patch_repository_locally_absent(repos.to_vec()).await?;
                     nurse_actions.append(&mut actions);
                 }
+                Defect::CoffeeGlobalrepoCleanup(networks) => {
+                    for (network, path) in networks {
+                        let global_repo = format!("{}/repositories", self.config.root_path);
+                        if !Path::exists(&Path::new(&path)) {
+                            move_dir_if_exist(&global_repo, &path).await?;
```

Comment:
here we are iterating over the networks to see if the repositories sub-directory exists and if it doesn't we move the global repository inside it.

but this way, in the second iteration, there is no `global_repo` and `move_dir_if_exist` will do nothing

Reply:
Good catch, we should copy it and remove the global repository at the end of the process, good catch